### PR TITLE
15th Bug Сrusade

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Fixed issue where feudal barons spawned spouses non-stop, but couldn't marry them.
- Fixed exploit where you could invite prestigious spouses of another incompatible with you race to your court and marry them to get prestige non-stop.
- Fixed issue where AI electors thought that all immortal or long-lived characters are too young to rule.
- Fixed issue where you couldn't use Burn Evil Being on courtiers.
- Fixed issue where undead couldn't pick Carousing focus, but could be invited to party. Now they can do both.
- Fixed issue where some historical children looked like adults.
- Fixed issue where race immortality didn't affect characters born after game start.
- Fixed issue where female gnomes, dwarves, elves (born after game start) became infertile when they reached 40 years old. Note for testers: you should save game and load to see that Infertile trait is added.
- Purged children with adult traits like personality traits (Cynical, Kind, Cruel, Ambitious), Poet and Celibate traits.
- Purged characters with both Kind and Cruel traits.
- Gave names to Warcraft songs.
- Fixed issue where blood elves could spawn before the Sunwell's Corruption.
- Fixed issue where you could click on Greatest Gift decision multiple times.
- Political Faction government no longer can use Force Vassalization CB on kings.

### CODE REVIEW

**Comment by MemeWizard** [comment]
>Purged children with adult traits like personality traits (Cynical, Kind, Cruel, Ambitious)

But children still have shrewd/brawny adult traits

>Poet

Not all are purged

<details>

image


</details>

**Comment by zumbak04** [comment]
> But children still have shrewd/brawny adult traits

Fixed!

> Not all are purged

Fixed!

**Comment by MemeWizard** [comment]
>Purged children with adult traits like personality traits (Cynical, Kind, Cruel, Ambitious), Poet and Celibate traits.

<details>

image

</details>

**Comment by geffe1** [comment]
I am unsure if we are meant to have the original CK2 tracks in the mod or not, either way- the names appear to show though some of them are named like this:
Frozen_Throne_Siege_Car_Something_Here

Unsure if that is meant to be the case, but I saw it and decided to at least make sure you all know about it.

File: history/characters/64000_goblin.txt [hunk 1]
```diff
@@ -233,7 +232,7 @@
 	dna="laehdcdmncj"
 	culture=goblin religion=cult_of_wealth
 	martial=7 diplomacy=6 stewardship=6 intrigue=8 learning=8
-	trait=grey_eminence trait = creature_goblin trait=cynical trait=paranoid trait=cruel trait=follower_of_nzoth trait=lunatic
+	trait=grey_eminence trait = creature_goblin trait=cynical trait=paranoid trait=follower_of_nzoth trait=lunatic
```
Comment (MemeWizard) [inline history/characters/64000_goblin.txt]: >trait=follower_of_nzoth

:thonk:

File: history/characters/120000_nraqi.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	martial=4 diplomacy=8 stewardship=4 intrigue=4 learning=5
 	add_trait=brilliant_strategist
 	add_trait=class_warrior_7
-	add_trait=humble add_trait=cruel add_trait=ambitious add_trait=lustful
+	add_trait=humble add_trait=ambitious add_trait=lustful add_trait=kind
```
Comment (MemeWizard) [inline history/characters/120000_nraqi.txt]: :thonk:

**Comment by zumbak04** [comment]
> I am unsure if we are meant to have the original CK2 tracks in the mod or not

It's fine.

> the names appear to show though some of them are named like this:
> Frozen_Throne_Siege_Car_Something_Here

REE, why you didn't take a screenshot?

**Comment by geffe1** [comment]
Everything seems ok, checked on the songs again as well and all of them are named like they should, so no underscores.

**Pull request merged**